ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.16.yaml

```diff
@@ -70,6 +70,32 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-preserve-bootstrap-ign
+- as: azure-ipi-marketplace-image-gen1-vm-type-gen1-f14
+  cron: 30 19 9,23 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_NP10s
+      CONTROL_PLANE_INSTANCE_TYPE: Standard_NP10s
+      CUSTOM_AZURE_REGION: southcentralus
+      OS_IMAGE_URN: RedHat:rh-ocp-worker:rh-ocp-worker-gen1:413.92.2023101700
+    test:
+    - chain: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-azure-ipi-marketplace-generation
```

Comment:
whether need previous cucushift-installer-rehearse-azure-ipi-vmgenv1  ?

Reply:
I think they are different scenarios, 
azure-ipi-marketplace-generation: use gen1 marketplace image and provision vm with only-gen1 supported instance type
azure-ipi-vmgenv1: provision vm with only-gen1 supported instance type, boot image comes from gallery image created by installer.